src/registrar/models/user_domain_role.py

```diff
@@ -15,7 +15,7 @@ class Roles(models.TextChoices):
         elsewhere.
         """
 
-        ADMIN = "manager"
+        MANAGER = "manager"
```

Comment:
**question**: I see this used in a variety of places in the code as `UserDomainRole.Roles.ADMIN`. If we change this to `MANAGER` here don't we need to change all the other occurrences to `UserDomainRole.Roles.MANAGER`? I think it's probably a better name for the role, but there might be more to change to match this refactoring?

Reply:
Yes I was adding this when this message came in, sorry for pushing too early. Resolving this and you should see the places where the mentions were changed.